exercises/practice/resistor-color-duo/src/test/java/ResistorColorDuoTest.java

```diff
@@ -1,4 +1,5 @@
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.Disabled;
```

Comment:
```suggestion
import org.junit.jupiter.api.Disabled;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;
```

Can you also move this up, so the imports are nicely arranged in a alphabetical order?